src/Raven.Client/Documents/Operations/GetAttachmentOperation.cs

```diff
@@ -135,6 +135,7 @@ public override void SetResponseUncached(HttpResponseMessage response, Stream st
                 var etag = response.GetRequiredEtagHeader();
                 IEnumerable<string> hashVal;
                 var hash = response.Headers.TryGetValues("Content-Hash", out hashVal) ? hashVal.First() : null;
+                var size = response.Content.Headers.ContentLength ?? 0;
```

Comment:
What are you reading here, and what if the request is using `Chunked-Encoding` ?